configuration/pih/htmlforms/hiv/zl/section-hiv-history.xml

```diff
@@ -713,11 +718,19 @@
                     <span class="small">
                         <obs conceptId="PIH:11949" showUnits="false"/>
                     </span>
-                </p>
 
-                <!-- Removed Secondhand smoke exposure -->
+                </p>
             </div>
 
+            <!-- Secondary smoke -->
```

Comment:
On the paper, this question is at the end of the section (not with smoking).  For now, we can keep it with smoking, but the order will be different from paper.  The Pediatrics form is also missing drugs.  I think that we should keep that for all ages.

Adult:  Smoking, Alcohol, Drugs
Pediatrics:  Smoking, Alcohol, Second-hand smoke